server/src/main/java/org/apache/iotdb/db/utils/HandleSystemErrorStrategy.java

```diff
@@ -18,11 +18,38 @@
  */
 package org.apache.iotdb.db.utils;
 
+import org.apache.iotdb.commons.cluster.NodeStatus;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public enum HandleSystemErrorStrategy {
   /** just set system status to error and then do nothing else */
   NONE,
   /** set system status to read-only and the system only accepts query operations */
   CHANGE_TO_READ_ONLY,
   /** the system will be shutdown */
-  SHUTDOWN,
+  SHUTDOWN;
+
+  private static final Logger logger = LoggerFactory.getLogger(HandleSystemErrorStrategy.class);
+
+  public NodeStatus handle() {
+    NodeStatus status = NodeStatus.Error;
+    if (this == HandleSystemErrorStrategy.NONE) {
+      logger.error(
+          "Unrecoverable error occurs! Just change system status to error when handle_system_error is NONE.",
+          new RuntimeException("System mode is set to ERROR"));
+    } else if (this == HandleSystemErrorStrategy.CHANGE_TO_READ_ONLY) {
+      logger.error(
+          "Unrecoverable error occurs! Change system status to read-only when handle_system_error is CHANGE_TO_READ_ONLY. Only query statements are permitted!",
```

Comment:
```suggestion
          "Unrecoverable error occurs! Change system status to read-only because handle_system_error is CHANGE_TO_READ_ONLY. Only query statements are permitted!",
```

Reply:
Fixed.